src/number/n_algExt.jl

```diff
@@ -120,11 +120,12 @@ function AbstractAlgebra.expressify(n::n_algExt; context = nothing)::Any
    end
 end
 
-function show(io::IO, n::n_algExt)
-   libSingular.StringSetS("")
-   libSingular.n_Write(n.ptr, parent(n).ptr, false)
-   m = libSingular.StringEndS()
-   print(io, m)
+function Base.show(io::IO, ::MIME"text/plain", a::n_algExt)
```

Comment:
This 3-arg method normally defaults to calling the 2-arg one (without MIME), can't this method be therefore deleted here?

Reply:
ok